packages/core/src/utils/anthropic-ai/index.ts

```diff
@@ -90,71 +90,115 @@ function addPrivateRequestAttributes(span: Span, params: Record<string, unknown>
 }
 
 /**
- * Add response attributes to spans
+ * Capture error information from the response
+ * @see https://docs.anthropic.com/en/api/errors#error-shapes
  */
-function addResponseAttributes(span: Span, response: AnthropicAiResponse, recordOutputs?: boolean): void {
-  if (!response || typeof response !== 'object') return;
+function handleResponseError(span: Span, response: AnthropicAiResponse): void {
+  if (response.error) {
+    const errorType = response.error.type || 'unknown_error';
+    span.setStatus({ code: SPAN_STATUS_ERROR, message: errorType });
 
-  // Private response attributes that are only recorded if recordOutputs is true.
-  if (recordOutputs) {
-    // Messages.create
-    if ('content' in response) {
-      if (Array.isArray(response.content)) {
-        span.setAttributes({
-          [GEN_AI_RESPONSE_TEXT_ATTRIBUTE]: response.content
-            .map((item: ContentBlock) => item.text)
-            .filter(text => !!text)
-            .join(''),
-        });
+    captureException(new Error(`anthropic_error: ${errorType}`), {
+      mechanism: {
+        handled: false,
+        type: 'auto.ai.anthropic',
+        data: {
+          function: 'anthropic_error',
+          error_message: response.error.message,
+        },
+      },
+    });
+  }
```

Comment:
m:  Two things here:

1. We should rather set a more specific `type` than adding `mechanism.data` because `mechanism.data` because `mechanism.type` is searchable why data is not:

<img width="694" height="72" alt="image" src="https://github.com/user-attachments/assets/0c4b59fa-cd7b-4f1b-81c2-af401b4d7711" />

2. What's the reason for creating a `new Error()` vs. directly capturing the `response.error`? Ideally, the error message can be part of the object we capture and isn't added as `mechanism.data.error_message`.  

So overall, I think something like this would produce a better error in Sentry but I might be missing a thing or two, so feel free to overrule me :D

```suggestion
    captureException(response.error), {
      mechanism: {
        handled: false,
        type: 'auto.ai.anthropic.anthropic_error',
      },
    });
  }
```